### PR TITLE
Code Style: Recommend initializing objects w properties

### PR DESCRIPTION
**Pull request by danrabbit**

### CODE REVIEW

File: writing-apps/code-style.md [hunk 1]
```diff
@@ -219,6 +219,29 @@ you should use
 can_focus = false;
 ```
 
+### Initialize Objects with Properites
+This is especially more clear when initializing an object with many properties. Avoid the following
+
+```csharp
```
Comment (tintou) [inline writing-apps/code-style.md:225]: highlightjs actually supports vala so you might want to switch to ```vala

File: writing-apps/code-style.md [hunk 2]
```diff
@@ -219,6 +219,29 @@ you should use
 can_focus = false;
 ```
 
+### Initialize Objects with Properites
+This is especially more clear when initializing an object with many properties. Avoid the following
```
Comment (jeremypw) [inline writing-apps/code-style.md:223]: Should be "clearer" not "more clear"

Comment (lewisgoddard) [inline writing-apps/code-style.md:223]:
```suggestion
This is especially clearer when initializing an object with many properties. Avoid the following
```

File: writing-apps/code-style.md (hunk 1 above)
Comment (lewisgoddard) [inline writing-apps/code-style.md:225]:
```suggestion
```vala
```

File: writing-apps/code-style.md [hunk 3]
```diff
@@ -219,6 +219,29 @@ you should use
...
+var label = new Gtk.Label ("Test Label");
+label.set_ellipsize (Pango.EllipsizeMode.END);
+label.set_valign (Gtk.Align.END);
+label.set_width_chars (33);
+label.set_xalign (0);
+```
+
+and instead do this
+
+```csharp
```
Comment (lewisgoddard) [inline writing-apps/code-style.md:235]:
```suggestion
```vala
```

**Pull request merged**